tools/certify/commands/generate-ticket.cpp

```diff
@@ -31,6 +31,7 @@ bool GenerateTicketCommand::parse(const std::vector<std::string>& opts)
         ("days", po::value<int>(&validity_days)->default_value(7), "Validity in days.")
         ("cam-permissions", po::value<std::string>(&cam_permissions), "CAM permissions as binary string (e.g. '1111111111111100' to grant all SSPs)")
         ("denm-permissions", po::value<std::string>(&denm_permissions), "DENM permissions as binary string (e.g. '000000000000000000000000' to grant no SSPs)")
+        ("include-gn-mgmt", "Include GN-MGMT's SSP (empty)")
```

Comment:
I guess it should be `gn-mgmt-permission` for consistency.

Reply:
I just thought `gn-mgmt-permission(s)` to be misleading because you cannot specify any permissions, but I don't mind.

Reply:
That's also true, maybe just `--gn-mgnt`?

Reply:
`--with-gn-mgmt` ?

Reply:
> `--with-gn-mgmt` ?

@kelunik, are you also okay with that?

Reply:
@glmax, don't really care. I'll let @riebl decide.

Reply:
I have no strong preference either, however, `--allow-gn-mgmt` would make it clear that it is related to permissions.

Reply:
I am totally convinced now :-)